docs/source/logging/logging.md

```diff
@@ -64,6 +64,10 @@ export COLUMNS=120 LINES=25
 You must provide a value for both `COLUMNS` and `LINES` even if you only wish to change the width of the log message. Rich's default values for these variables are `COLUMNS=80` and `LINE=25`.
 ```
 
+### Rich logging in Jupyter
+
+Rich also formats the logs in JupyterLab and Jupyter Notebook. The size of the output console does not adapt to your window but can be controlled through the `JUPYTER_COLUMNS` and `JUPYTER_LINES` environment variables, which default to 115 and 100 respectively.
```

Comment:
So what's the "ideal" setting for these environment variables is it 115 and 100 or something else?

Reply:
115 and 100 seemed to be ideal, so I don't think many people will want to change them. That's what's shown in the screenshot.

Reply:
Okay, so just to clarify do they need to do anything when they want to use Rich in Jupyter? It's not super clear to me now if they need to set those environment variables or not.

Reply:
No, they don't need to do anything. They only need to change the environment variables if they don't like the default width that seemed to work well for me. Let me make this clearer!

Reply:
```suggestion
Rich also formats the logs in JupyterLab and Jupyter Notebook. The size of the output console does not adapt to your window but can be controlled through the `JUPYTER_COLUMNS` and `JUPYTER_LINES` environment variables. The default values (115 and 100 respectively) should be suitable for most users, but if you require a different output console size then you should alter the values of `JUPYTER_COLUMNS` and `JUPYTER_LINES`.
```

Reply:
@MerelTheisenQB how's that suggestion?

Reply:
Great, that's very clear to me now 🙂